### PR TITLE
Adding ProfitBricks as a ComputeService provider.

### PR DESCRIPTION
**Pull request by ericknauer**
Added ProfitBricks and related documentation.

### CODE REVIEW

**Comment by jclouds-commentator** [comment]
Go to http://951be72bf9f96507ee61-1045708971e5c88e33b8c4a601bf51a0.r28.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://e1cf4fa00940bf4c7d80-bd3d2d593cc1f42c168edc42adcf75e1.r97.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://0eb09892d6dc7ac4cc72-1d27093a324170395626955656da459c.r32.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://04e2e7e2f8c579fe4081-ef476e2cecd28cb69148198b9bd69d03.r27.cf5.rackcdn.com/ to review your changes.

File: guides/profitbricks.md [hunk 1]
```diff
@@ -0,0 +1,476 @@
...
+			/*
+			 * Fetches list of all Servers
+			 */
+			List<Server> servers = api.serverApi().getAllServers();
+
+			api.dataCenterApi().deleteDataCenter(dc.id());
+		}
+	}
+	
+## pom.xml
```
Comment (nacx) [inline guides/profitbricks.md]: Why is the provider pom here? This is the pom for the profitbricks provider, but not the one users would use to build their apps using it.
I'd just remove this pom.

**Comment by nacx** [comment]
Thanks @ericknauer! This is great. Mind adding also an example of how to use ProfitBricks with the portable abstraction? That's what users are looking for when coming to jclouds. A copy/paste of [the README](https://github.com/jclouds/jclouds-labs/tree/master/profitbricks) would be fine.

**Comment by jclouds-commentator** [comment]
Go to http://f48af4014050f4fae835-4fce1bcea9f919da61306ad78961afce.r70.cf5.rackcdn.com/ to review your changes.

**Comment by ericknauer** [comment]
Hello Ignasi,

Thanks for the feedback. I've attempted to implement the changes you
suggested.

Eric

**Comment by jclouds-commentator** [comment]
Go to http://758fb7e57ce8f62fb68b-07b9ff802ca96ecda87349f49ec43649.r69.cf5.rackcdn.com/ to review your changes.

**Comment by nacx** [comment]
Thanks @ericknauer! Just a few final comments:

* Put the abstraction tutorial the first one, as it is the common case in jclouds.
* Mark ProfitBricks with an `*` in the providers list to indicate it is still a "labs" provider (see how other providers in labs appear in that list).
* Add a link tot he profitbricks provider in the [user guide index file](https://github.com/jclouds/jclouds-site/blob/master/guides/index.md).
* Enclose the java code inside the `{% highlight java %}` and `{% endhighlight %}` tags so the code is properly formatted and easier to read.

**Comment by jclouds-commentator** [comment]
Go to http://17bac6c3388c0d099d72-6ae2988f6435681dbfa1be7f7aa26b75.r25.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://031560c55db8d4a4372c-f4bb1f71fe6dc16540befa05975dea0e.r18.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://843af3d8f18d46bcbc3e-5d646d7e93f2f624295578bc558f1e41.r21.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://392d1541d91197ae3aa2-2f7a6a07fb046addf93239bb6d9fc812.r38.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://690cc2f1997576d8b255-2865546ad9ea1bd071cee9d52001cbca.r30.cf5.rackcdn.com/ to review your changes.

**Comment by ericknauer** [comment]
Hi Ignasi,

I made quite a few changes as you requested. I was having a bit of issue
with the {% highlight java %} tags, but hopefully got it working the way
you want. It appears that github uses a different markdown parser than the
jclouds site. :-)

Wrapping the java code sections with:

```java

```

made it look nice in the github preview, but didn't work like the {%
highlight java %} tags.

Thanks,

Eric

File: guides/profitbricks.md [hunk 2]
```diff
@@ -0,0 +1,443 @@
...
+-  Login to your server, and change the password, as requested.
+
+
+    ~ ssh root@<remote-ip>
+    ...
+    Changing password for root.
+    (current) UNIX password: 
+    Enter new UNIX password: 
+    Retype new UNIX password: 
+    ~ root@ubuntu:~# exit
```
Comment (nacx) [inline guides/profitbricks.md]: Also wrap this inside a ``` tag so ti is properly rendered?

**Comment by nacx** [comment]
Many thanks @ericknauer! This looks great! I've just noted one thing about a block that is not properly displayed. Mind fixing that? I'll merge the PR and publish the site as soon as it's fixed.
Thx!

**Comment by jclouds-commentator** [comment]
Go to http://9a27c61b55f02527d379-5d2190bf794a50557136f6e9a514d71a.r38.cf5.rackcdn.com/ to review your changes.

**Comment by nacx** [comment]
I've squashed the commits into a single one and added a small one that fixed the links in the ToC.
Merged and published to the site. Thanks @ericknauer!

**Comment by ericknauer** [comment]
Thanks for working through the whole process with me!

Eric

**Pull request closed without merging**